Copy improvements in the SQL Database verified source (#749)
dlt-hub/dlt: docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

> # +30 SQL Databases

Reviewer: ```suggestion
# 30+ SQL Databases
```

SQL databases are management systems (DBMS) that store data in a structured format, commonly used
for efficient and reliable data retrieval.

Our SQL Database verified source loads data to your specified destination using SQLAlchemy.

> > View the pipeline example

Reviewer: I suggest using a tip [admonition](https://docusaurus.io/docs/markdown-features/admonitions) instead of the markdown blockquote.

| sql_database | Retrieves data from an SQL database       |
| sql_table    | Retrieves data from an SQL database table |

### Supported databases

We support all [SQLAlchemy dialects](https://docs.sqlalchemy.org/en/20/dialects/), which include, but are not limited to, the following database engines:

* PostgreSQL
* MySQL
* SQLite
* Oracle
* Microsoft SQL Server
* MariaDB
* IBM DB2 and Informix
* Google BigQuery
* Snowflake
* Redshift
* Apache Hive and Presto
* SAP Hana
* CockroachDB
* Firebird
* Teradata Vantage

> > Note that there many unofficial dialects, such as [DuckDB](https://duckdb.org/).

Reviewer: Same here: it's a good place to use a note [admonition](https://docusaurus.io/docs/markdown-features/admonitions) instead of the markdown blockquote.

## Setup Guide

### Grab credentials

This verified source utilizes SQLAlchemy for database connectivity. Let's take a look at the following public database example:

`connection_url = "mysql+pymysql://rfamro@mysql-rfam-public.ebi.ac.uk:4497/Rfam"`

> > The database above doesn't require a password.

Reviewer: ```suggestion
The database above doesn't require a password.
```

   dlt init sql_database duckdb
   ```

   It will initialize
   [the pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/sql_database_pipeline.py)
   with an SQL database as the [source](../../general-usage/source) and
   [DuckDB](../destinations/duckdb.md) as the [destination](../destinations).

>    > If you'd like to use a different destination, simply replace `duckdb` with the name of your preferred [destination](../destinations).

Reviewer: That's also a good place to use a tip admonition.

   pip install -r requirements.txt
   ```

1. Run the verified source by entering:

   ```bash
   python sql_database_pipeline.py
   ```

1. Make sure that everything is loaded as expected with:

   ```bash
   dlt pipeline <pipeline_name> show
   ```

>    > The pipeline_name for the above example is `rfam`, you may also use any

Reviewer: It's best to either remove a blockquote or use a "note" admonition here.

   info = pipeline.run(source, write_disposition="merge")
   print(info)
   ```

>    > In this example, we load data from the 'family' table, using the 'updated' column for incremental loading. In the first run, the process loads all data starting from midnight (00:00:00) on January 1, 2022. Subsequent runs perform incremental loading, guided by the values in the 'updated' field.

Reviewer: ```suggestion
   In this example, we load data from the `family` table, using the `updated` column for incremental loading. In the first run, the process loads all data starting from midnight (00:00:00) on January 1, 2022. Subsequent runs perform incremental loading, guided by the values in the `updated` field.
```

   print(info)
   ```

>    > This process initially loads all data from the "family" table starting at midnight on January 1, 2022. For later runs, it uses the "updated" field for incremental loading as well. 

Reviewer: ```suggestion
   This process initially loads all data from the `family` table starting at midnight on January 1, 2022. For later runs, it uses the `updated` field for incremental loading as well. 
```
